ruby/ql/lib/codeql/ruby/frameworks/http_clients/Excon.qll

```diff
@@ -7,6 +7,7 @@ private import codeql.ruby.CFG
 private import codeql.ruby.Concepts
 private import codeql.ruby.ApiGraphs
 private import codeql.ruby.DataFlow
+private import codeql.ruby.dataflow.internal.DataFlowImplForLibraries as DataFlowImplForLibraries
```

Comment:
A `DataFlowImpl` copy should only be used multiple times in library code, when we can guarantee that either (a) those copies will _never_ be evaluated together (unlikely), or (b) that they are _always_ evaluated together. Both can be difficult to guarantee, so it is often easier to simply add more copies of `DataFlowImpl` (e.g. `DataFlowImplForLibraries2`, `DataFlowImplForLibraries3`, etc.

Reply:
Done in 528ef0e as discussed offline